### ISSUE TITLE
Upstream security vulnerability markdown-it-katex https://www.npmjs.com/advisories/1466

### ISSUE THREAD

**Issue by EEParker** [quasarframework/quasar-ui-qmarkdown#120]
Hi! I really love this plugin, but this security vulnerability prevents from being able to use it. I'm not sure if this module is something that can be replaced, forked or dropped but I would appreciate any feedback. 

It looks like there are a few open MRs to fix the issue, but the project seems to be dead.

See https://github.com/waylonflinn/markdown-it-katex/pull/31/files

## Security Advisory
https://www.npmjs.com/advisories/1466

```
  High            Cross-Site Scripting                                          
                                                                                
  Package         markdown-it-katex                                             
                                                                                
  Patched in      No patch available                                            
                                                                                
  Dependency of   @quasar/quasar-app-extension-qmarkdown [dev]                  
                                                                                
  Path            @quasar/quasar-app-extension-qmarkdown >                      
                  @quasar/quasar-ui-qmarkdown > markdown-it-katex               
                                                                                
  More info       https://npmjs.com/advisories/1466   
```

## Overview
All versions of markdown-it-katex are vulnerable to Cross-Site Scripting (XSS). The package fails to properly escape error messages, which may allow attackers to execute arbitrary JavaScript in a victim's browser by triggering an error.

## Remediation
No fix is currently available. Consider using an alternative package until a fix is made available.

## Resources
- [GitHub issue](https://github.com/waylonflinn/markdown-it-katex/issues/26)

**Comment by hawkeye64** [comment]
@EEParker Thanks for letting me know about this. Originally, the idea was to integrate Katex, but it never happened. The code does not use it at all. The dependency exists only in package.json, so current QMarkdown versions are still good. I will remove the unused dependency, which should get rid of the security advisory.

**Comment by hawkeye64** [comment]
@EEParker Fixed in v1.0.32

**Comment by EEParker** [comment]
Thank you for the prompt response and fixing this so quickly!
